src/Pagerfanta/Adapter/SphinxAdapter.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/*
+ * This file is part of the Pagerfanta package.
+ *
+ * (c) Maikel Doezé <maikel.doeze@marqin.nl>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+ 
+namespace Pagerfanta\Adapter;
+
+use SphinxClient;
+
+class SphinxAdapter implements AdapterInterface
+{
+	private $client;
+	private $query;
+    private $index;
+    private $comment;
+    private $results;
+    private $maxMatches = 0;
+    private $cutoff = 0;
+	
+	/**
+	 * Constructor.
+	 *
+	 * @param SphinxClient $client A Sphinx client.
+	 * @param $query A Sphinx query.
```

Comment:
Are `@query`, `$index` and `$comment` all strings?  Please could you update the docblock to show this?